Update the contributing docs for clarity and to encourage community activity (#9035)
traefik/traefik: docs/content/contributing/submitting-issues.md

# Submitting Issues

> ### Help Us Help You!

Reviewer: ```suggestion
Help Us Help You!
```

{: .subtitle }

> Issues are perfect for requesting a feature/enhancment or reporting a suspected bug. We use the [GitHub issue tracker](https://github.com/traefik/traefik/issues) to keep track of issues in Traefik. 

Reviewer: ```suggestion
Issues are perfect for requesting a feature/enhancement or reporting a suspected bug.
We use the [GitHub issue tracker](https://github.com/traefik/traefik/issues) to keep track of issues in Traefik. 
```

The process of sorting and checking the issues is a daunting task, and requires a lot of work (more than an hour a day ... just for sorting).

> To help us (and other community members) quickly and easily understand what you need, be sure to follow the guidelines below. 

Reviewer: ```suggestion
To help us (and other community members) quickly and easily understand what you need,
be sure to follow the guidelines below. 
```

!!! important "Getting Help Vs Reporting an Issue"

    The issue tracker is not a general support forum, but a place to report bugs and asks for new features.

    For end-user related support questions, try using the [Traefik Community Forum] (https://community.traefik.io/)
 [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Discourse)](https://community.traefik.io/)

## Issue Title

The title must be short and descriptive. (~60 characters)

Examples:
          

>           * Bug: Duplicate requests in access logs
>           * FR:  Support fastCGI 

Reviewer: ```suggestion
* Bug: Duplicate requests in access logs
* Feature: Support TCP
```

            
            
## Feature Request

Traefik is an open source project and aims to be the best edge router possible. 

> Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups). Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.

Reviewer: ```suggestion
Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups).
Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.
```

> Do your best to explain what you're looking for, and why it would improve Traefik for everyone. Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible. Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) naturally rise to the top of our to-do list.  

Reviewer: ```suggestion
Do your best to explain what you're looking for, and why it would improve Traefik for everyone.
Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible.
Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) naturally rise to the top of our to-do list.  
```

> If you are interested in creating a PR for your feature request, let us know in the issue.  It can take a lot of work to make sure a PR can integrate with our existing code and planning with the team ahead of time can make sure that your PR can be accepted and merged quickly.

Reviewer: ```suggestion
If you are interested in creating a PR for your feature request, let us know about the issue.  
It can take a lot of work to make sure a PR can integrate with our existing code and planning with the team ahead of time can make sure that your PR can be accepted and merged quickly.
```

## Issues or Possible Bug Reports

Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.

> Explain the conditions in which you encountered the issue; what is your context? Share any logs you may have and make sure to share the steps it takes to reproduce your issue or bug. 

Reviewer: ```suggestion
Explain the conditions in which you encountered the issue; what is your context?
Share any logs you may have and make sure to share the steps it takes to reproduce your issue or bug. 
```

Help Us Help You!
{: .subtitle }

Issues are perfect for requesting a feature/enhancement or reporting a suspected bug.
We use the [GitHub issue tracker](https://github.com/traefik/traefik/issues) to keep track of issues in Traefik. 

The process of sorting and checking the issues is a daunting task, and requires a lot of work (more than an hour a day ... just for sorting).
To help us (and other community members) quickly and easily understand what you need,

> be sure to follow the guidelines below. 

Reviewer 2: ```suggestion
be sure to follow the guidelines below.
```

!!! important "Getting Help Vs Reporting an Issue"

    The issue tracker is not a general support forum, but a place to report bugs and asks for new features.

>     For end-user related support questions, try using the [Traefik Community Forum] (https://community.traefik.io/)

Reviewer 2: ```suggestion
    For end-user related support questions, try using the [Traefik Community Forum](https://community.traefik.io/)
```

>     For end-user related support questions, try using the [Traefik Community Forum] (https://community.traefik.io/)
>  [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Discourse)](https://community.traefik.io/)

Reviewer 2: ```suggestion
    For end-user related support questions, try using the [Traefik Community Forum](https://community.traefik.io/)
   [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Traefik%20Community%20Forum)](https://community.traefik.io/)
```

## Issue Title

The title must be short and descriptive. (~60 characters)

Examples:

> * Bug: Duplicate requests in access logs

Reviewer 2: ```suggestion
* Handle TLS for unknown names correctly ([#8888](https://github.com/traefik/traefik/issues/8888))
```

> * Feature: Support TCP

Reviewer 2: ```suggestion
* Traefik Should Healthcheck HTTP2 Connections to Backends ([#8375](https://github.com/traefik/traefik/issues/8375))
```

## Feature Request

Traefik is an open source project and aims to be the best edge router possible. 

Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups).

> Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.

Reviewer 2: ```suggestion
Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE/feature-request.yml) as much as possible.
```

Do your best to explain what you're looking for, and why it would improve Traefik for everyone.
Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible.
Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) indicate higher community interest and help us to prioritize.  

If you are interested in creating a PR for your feature request, let us know in the the issue so we can work with you.  
It can take a lot of work to make sure a PR can integrate with our existing code and planning with the team ahead of time can make sure that your PR can be accepted and merged quickly.

## Issues or Possible Bug Reports

> Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.

Reviewer 2: ```suggestion
Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml) as much as possible.
```

    The issue tracker is not a general support forum, but a place to report bugs and asks for new features.

    For end-user related support questions, try using the [Traefik Community Forum] (https://community.traefik.io/)
 [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Discourse)](https://community.traefik.io/)

## Issue Title

The title must be short and descriptive. (~60 characters)

Examples:
          
* Bug: Duplicate requests in access logs
* Feature: Support TCP
            
            
## Feature Request

Traefik is an open source project and aims to be the best edge router possible. 

Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups).
Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.

Do your best to explain what you're looking for, and why it would improve Traefik for everyone.
Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible.

> Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) naturally rise to the top of our to-do list.  

Author: ```suggestion
Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) indicate higher community interest and help us to prioritize.  
```

 [![Join the chat at https://community.traefik.io/](https://img.shields.io/badge/style-register-green.svg?style=social&label=Discourse)](https://community.traefik.io/)

## Issue Title

The title must be short and descriptive. (~60 characters)

Examples:
          
* Bug: Duplicate requests in access logs
* Feature: Support TCP
            
            
## Feature Request

Traefik is an open source project and aims to be the best edge router possible. 

Remember when asking for new features that these must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups).
Follow the [issue template](https://github.com/traefik/traefik/blob/master/.github/ISSUE_TEMPLATE.md) as much as possible.

Do your best to explain what you're looking for, and why it would improve Traefik for everyone.
Be detailed and share the use-case(s) to allow us to see the value of your feature request as quickly as possible.
Features with a lot of positive interaction (claps, +1s, conversation about how this would impact them) naturally rise to the top of our to-do list.  

> If you are interested in creating a PR for your feature request, let us know about the issue.  

Author: ```suggestion
If you are interested in creating a PR for your feature request, let us know in the the issue so we can work with you.  
```

Author: @ldez, is this clear?
